packages/massa-web3/src/experimental/basicElements/operationManager.ts

```diff
@@ -0,0 +1,286 @@
+import { unsigned } from 'big-varint'
```

Comment:
I think this should be abstracted to not depend from it directly there (we should be able to use the serialize interface)

Reply:
Varint is intrinsically part of the serialization and deserialization logic. If we want to offer abstraction, the part that should be abstracted should be the `Canonicalizer` (canonicalization operation including `canonicalize` and `uncanonicalization`, if that makes sense from a code usability perspective).

Reply:
Yes I agree no need to change for now

Reply:
See #564